Detectors/TRD/base/macros/PrintRun2DigitsAndRun3

```diff
@@ -0,0 +1,112 @@
+#if !defined(__CINT__) || defined(__MAKECINT__)
+#include <TClonesArray.h>
+#include <TEveLine.h>
+#include <TEveManager.h>
+#include <TEveElement.h>
+
+#include <AliRunLoader.h>
+#include <AliLoader.h>
+#include <AliDataLoader.h>
+#include <AliTreeLoader.h>
+#include <AliTRDarrayADC.h>
+#include <AliTRDtrackletWord.h>
+#include <AliTRDtrackletMCM.h>
+#include <AliEveEventManager.h>
+#include <AliEveTRDData.h>
+#endif
+
+#include <iostream>
+
+#include "TH1F.h"
+#include "AliTRDtrackletMCM.h"
+//#include "TRDDataFormat/TriggerRecord.h"
+#include "TRDBase/Digit.h"
+#include "TRDBase/Tracklet.h"
+
+using namespace o2;
+using namespace trd;
+using namespace std;
+
+
+void ConvertRun2DigitsAndTracklets()
```

Comment:
this function name is the same as in another macro above. Is this intented?

Reply:
this one should actually not be there and is now removed, and rested added to the ExclusionList